frontend/src/app/core/components/toast-notification/toast-notification.component.html

```diff
@@ -1 +1,3 @@
-<p-toast></p-toast>
+<div class="global-toast">
+    <p-toast [baseZIndex]="9999"></p-toast>
```

Comment:
Also increased base index because this: 
![image](https://user-images.githubusercontent.com/36418695/129281429-a3b1a194-a854-4bff-ae8f-b894c1f094f7.png)